From 8802af5a85d6a41d6dade70da20ca8c54ad8c681 Mon Sep 17 00:00:00 2001
From: Ross Elliot <Ross.Elliot@ess.eu>
Date: Mon, 4 Jan 2021 09:03:43 +0100
Subject: [PATCH] [PATCH] Set CONFIG_FSL_IFC default y

Otherwise it is disabled if ifc nand is disabled but we also
have the PON fpga on the same bus.
---
 drivers/memory/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/memory/Kconfig b/drivers/memory/Kconfig
index 9bddca292330..e54c222f2b67 100644
--- a/drivers/memory/Kconfig
+++ b/drivers/memory/Kconfig
@@ -125,6 +125,7 @@ config FSL_CORENET_CF
 
 config FSL_IFC
 	bool
+	default y
 	depends on FSL_SOC || ARCH_LAYERSCAPE || SOC_LS1021A || COMPILE_TEST
 	depends on HAS_IOMEM
 
-- 
2.20.1

